manager/assets/modext/widgets/core/modx.panel.js

```diff
@@ -477,59 +477,6 @@ Ext.extend(MODx.FormPanel,Ext.FormPanel,{
         }
     }
 
-    /**
-     * @property {Function} insertTagCopyUtility - Updates placeholder tag in element name's help
-     * field to the current element name and attaches a listener to copy the tag when clicked on
-     *
-     * @param {Object} cmp - The help field's Ext.Component object
-     * @param {String} elType - The MODX element type (i.e., tv, chunk, or snippet)
-     */
-    ,insertTagCopyUtility: function(cmp, elType) {
```

Comment:
Moving this looks like a breaking change. Consider keeping the method but making it pass through to `MODx.util.insertTagCopyUtility`.

Reply:
Ok, will do in the next commit.